en/docs/GettingStarted/overview.md

```diff
@@ -45,26 +45,26 @@ WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Pub
 
 The diagram above shows the common lifecycle activities of an API developer/manager.
 
--   #### Developer Portal
+#### Developer Portal
 
 ![]({{base_path}}/assets/attachments/103327648/126556772.png)
 
-The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self register](https://docs.wso2.com/display/AM260/Customizing+the+API+Store) , discover, evaluate, subscribe to and consume APIs.
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self register](../Learn/ConsumeAPI/Customizations/customizing-the-api-store.md) , discover, evaluate, subscribe to and consume APIs.
```

Comment:
Normally doc links are added as follows without .md
[Configuring API Manager Analytics](../../../../Learn/Analytics/configuring-apim-analytics/)
Pls check if this link works

Reply:
This works locally. And there are a couple of other pages with the same type of links

Reply:
when you have .md, in the URL how does it show?